### PR TITLE
SVY-272: Add survey standard report retrieval to sdk

### PR DESCRIPTION
**Pull request by BnSmth**

### CODE REVIEW

File: src/main/java/esendex/sdk/java/parser/RecipientDataConverter.java [hunk 1]
```diff
@@ -0,0 +1,53 @@
...
+import com.thoughtworks.xstream.io.HierarchicalStreamReader;
+import com.thoughtworks.xstream.io.HierarchicalStreamWriter;
+
+import java.util.AbstractMap;
+import java.util.HashMap;
+import java.util.Map;
+
+public class RecipientDataConverter implements Converter {
+
+    public boolean canConvert(Class clazz) {
```
Comment (SLStinson) [inline src/main/java/esendex/sdk/java/parser/RecipientDataConverter.java]: Clazz?

File: src/test/java/com/esendex/java/sdk/Surveys/SurveySendService/SendWithMetaDataTests.java [hunk 2]
```diff
@@ -0,0 +1,67 @@
...
+    private static StubMethod stubMethod;
+    private static String expectedRequestBody;
+
+    @BeforeClass
+    public static void beforeClass() throws EsendexException {
+
+        expectedRequestBody = "<?xml version=\"1.0\" encoding=\"UTF-8\"?>" +
+                "<recipients xmlns=\"http://api.esendex.com/ns/\">" +
+                "<recipient><phonenumber>07123456789</phonenumber>" +
+                "<templatefields/>" +
```
Comment (SLStinson) [inline src/test/java/com/esendex/java/sdk/Surveys/SurveySendService/SendWithMetaDataTests.java]: This tag isn't needed, right?

File: src/test/java/com/esendex/java/sdk/Surveys/SurveySendService/SendWithTemplateFieldsTests.java [hunk 3]
```diff
@@ -0,0 +1,67 @@
...
+    private static String expectedRequestBody;
+
+    @BeforeClass
+    public static void beforeClass() throws EsendexException {
+
+        expectedRequestBody = "<?xml version=\"1.0\" encoding=\"UTF-8\"?>" +
+                "<recipients xmlns=\"http://api.esendex.com/ns/\">" +
+                "<recipient><phonenumber>07123456789</phonenumber>" +
+                "<templatefields><templatefield><name>City</name><value>Nottingham</value></templatefield></templatefields>" +
+                "<metadata/>" +
```
Comment (SLStinson) [inline src/test/java/com/esendex/java/sdk/Surveys/SurveySendService/SendWithTemplateFieldsTests.java]: This element isn't needed?

File: src/main/java/esendex/sdk/java/ServiceFactory.java [hunk 4]
```diff
@@ -82,11 +82,19 @@ public ContactService getContactService() {
 	}
 
 	/**
-	 * Obtains a service suitable for operations on the surveys resource.
+	 * Obtains a service suitable for sending surveys to new recipients.
 	 * @return the service
 	 */
-	public SurveysService getSurveysService() {
-		return new SurveysServiceImpl(authenticator);
+	public SurveySendService getSurveySendService() {
```
Comment (jamesturner) [inline src/main/java/esendex/sdk/java/ServiceFactory.java]: 👍

**Pull request merged**